sfs/mono/__init__.py

```diff
@@ -5,11 +5,56 @@
 
     drivingfunction
     source
-    synthesized
     soundfigure
 
 """
+import numpy as _np
+
 from . import drivingfunction
 from . import source
-from . import synthesized
 from . import soundfigure
+
+from .. import array as _array
+
+
+def shiftphase(p, phase):
+    """Shift phase of a sound field."""
+    p = _np.asarray(p)
+    return p * _np.exp(1j * phase)
+
+
+def synthesize(d, weights, ssd, secondary_source_function, **kwargs):
```

Comment:
The variable name `array` is mostly used in other codes.
Is there a reason using `ssd` here?